### PR TITLE
Login styles changed

### LINKED ISSUE #11: CHANGES IN THE LOGIN
### Explanation 
Input is a box for entering text. In this case, it directly concerns the user's email or password, although the input can also carry out other diverse actions, depending on what function it will have on the web page. This input is white with the code #FFFFFF, which makes the letters of any color stand out except for white, since the letters would not be visible. The purpose of the login button is that at the time of activating it, the person can enter from their user after having placed their username and password in the input. Now the button is blue with code #19456B and letters in a white tone with code #FFFFFF, unlike the previous design, this makes it more visually attractive for the user or person who enters the web page.
### Screenshots
image

### Checklist

- [ ] The button (login) has a slightly dark blue color of code #19456B and the inner text have a white tone #FFFFFF
- [ ] The "Iniciar sesion" text have a color:
- [ ] "¿Has olvidado tu contraseña" now have a terciary color
- [ ] The entrance has a blue border, color type
complementary with code #11698E
- [ ] The entrance has a gray background, color type complementary with code #11698E

### References
https://www.figma.com/file/zzEwuPfGTiZh4EA11MyZ5c/Sencico?node-id=65%3A51

### PR DESCRIPTION
**Pull request by e-azocar**
I added the new styles to the Login page, in the `tailwind.config.js` I defined the colors to use in the app as new classnames.

fixes #11 

## Screenshots 
image

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
+      "7xl": ["4rem", "1"],
+      "8xl": ["6rem", "1"],
+      "9xl": ["8rem", "1"],
+    },
+    colors: {
+      white: "#FFFFFF",
+      black: "#000000",
+      primary: "#19456B",
+      secondary: "#96CEB4",
+      terciary: "#11698E",
```
Comment (Katsu08) [inline tailwind.config.js:24]: change please for "tertiary"

**Review by Katsu08 — CHANGES_REQUESTED** [review]
I tested on local and doesn't look like the description? . I was cloned the repo and install dependencies, move to the branch then started it. And when I go to Login, I still see the background red.

**Review by Katsu08 — APPROVED** [review]
It looks good! But I notice that the navbar has not changed. Can you change the navbar button color to blue? To look like the design. Or it is for other issue?

image

**Review by Katsu08 — APPROVED** [review]
I see it good. Thanks!

**Comment by angelik0828** [comment]
I see it well!!!

**Pull request merged**